src/context/android-v1.js

```diff
@@ -0,0 +1,15 @@
+const common = require('./shared')
+const { getPlatformClasses } = require('../schema')
+
+const androidMapping = getPlatformClasses('android').reduce(
+  (androidMapping, { id: className }) => {
```

Comment:
Why are you calling `id` a `className`? Please add a comment explaining what's happening here.